documentation/config_file_puppetserver.markdown

```diff
@@ -46,6 +46,12 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 
     * `use-legacy-auth-conf`: Optional. The method to be used for authorizing access to the HTTP endpoints served by the master service. The applicable endpoints are listed in [Puppet v3 HTTP API](https://docs.puppet.com/puppet/latest/reference/http_api/http_api_index.html#puppet-v3-http-api).
 
+            If this setting is set to `true` or is not specified, Puppet uses the [deprecated][] Ruby `puppet-agent` authorization method and [Puppet `auth.conf`][`auth.conf` documentation] format, which will be removed in a future version of Puppet Server.
+
+            For a value of `false`, Puppet uses the HOCON configuration file format and location.
+
+            For more information, see the [`auth.conf` documentation][].
+
```

Comment:
Looks like these lines are indented too far. They're showing up as a code block in the "rich diff" mode